src/items/functions.md

```diff
@@ -109,34 +109,70 @@ sufficient context to determine the type parameters. For example,
 
 ## Extern functions
 
-Extern functions are part of Rust's foreign function interface, providing the
-opposite functionality to [external blocks]. Whereas external
-blocks allow Rust code to call foreign code, extern functions with bodies
-defined in Rust code _can be called by foreign code_. They are defined in the
-same way as any other Rust function, except that they have the `extern`
-qualifier.
+Extern function _definitions_ allow defining functions that can be called 
+with a particular ABI:
+
+```rust,norun
+extern "ABI" fn foo() { ... }
```

Comment:
There's a CI error here.  I would just show a simple example like:

```suggestion
extern "C" fn foo() { /*...*/ }
```

Reply:
@ehuss I think using `"ABI"` here is better for exposition purposes. I tried to avoid running this with `norun`, but that did not work out. I'll fix that.

Reply:
This should be fixed in the latest commit, so I'm resolving this issue. 

Please feel free to re-open if you disagree with using `"ABI"` for exposition purposes.